spec/stack.js

```diff
@@ -21,35 +21,49 @@ describe('Stack', () => {
   })
 
   context('pop()', () => {
-    it('returns and removes the top element in a stack or null if stack is empty', () => {
-      const myStack = new Stack()
+    context('when the stack is not empty', () => {
+      it('returns and removes the top element in a stack', () => {
```

Comment:
Consider adding a test that tests the top value as well (you're currently only testing that the size changes, not that you're getting the `top` element in the stack).